src/modules/Builder/utils/getUpdatedActivityFlows.ts

```diff
@@ -0,0 +1,18 @@
+import { ActivityFlowFormValues } from 'modules/Builder/types';
+
+export const getUpdatedActivityFlows = (
+  activityFlows: ActivityFlowFormValues[],
+  keyToRemove: string,
+) =>
+  activityFlows.reduce((acc: ActivityFlowFormValues[], flow) => {
```

Comment:
could you a bit simplified the code to keep the reference as it was:
`keyToRemove ? (...) : activityFlows`

Reply:
Could you please provide more details?